tox.ini

```diff
@@ -37,7 +37,7 @@ deps =
     mypy-extensions
     types-requests
     types-PyYAML
-commands = mypy --strict src --implicit-reexport --ignore-missing-imports --disable-error-code misc
+commands = mypy --strict --install-types --non-interactive src --implicit-reexport --ignore-missing-imports --disable-error-code misc
```

Comment:
What does this addition to tox do?

Reply:
installs mypy extensions for any initally unsupported package (in this case dateutils). but if it's ugly i can remove it and edit my code to workaround

Reply:
roger